erdpy/proxy/cost.py

```diff
@@ -0,0 +1,80 @@
+import base64
+
+from erdpy.proxy.tx_types import TxTypes
+from erdpy import utils, errors
+from erdpy.projects import load_project
+
+
+def prepare_argument(argument):
+    hex_prefix = "0X"
+    as_string = str(argument).upper()
+
+    if as_string.startswith(hex_prefix):
+        return as_string[len(hex_prefix):]
+
+    if not as_string.isnumeric():
+        raise errors.UnknownArgumentFormat(as_string)
+
+    as_number = int(as_string)
+    as_hexstring = hex(as_number)[len(hex_prefix):]
+    if len(as_hexstring) % 2 == 1:
+        as_hexstring = "0" + as_hexstring
+
+    return as_hexstring
+
+
+class TransactionCostEstimator:
+    def __init__(self, proxy_url):
+        self.proxy = proxy_url
+
+    def estimate_tx_cost(self, arguments):
+        tx_type = arguments.tx_type
+
+        if tx_type == TxTypes.MOVE_BALANCE:
+            self.__estimate_move_balance(arguments.data)
+        elif tx_type == TxTypes.SC_DEPLOY:
+            self.__estimate_sc_deploy(arguments.path)
+        else:
+            self.__estimate_sc_call(arguments.sc_address, arguments.function, arguments.arguments)
+
+    def __estimate_move_balance(self, data):
```

Comment:
One underscore, also for functions below.

Reply:
done.